packages/legend-query/src/stores/QueryBuilderState.ts

```diff
@@ -263,11 +268,26 @@ export class QueryBuilderState {
     this.resetQueryBuilder();
     this.resetQuerySetup();
     if (!rawLambda.isStub) {
+      const lambdaJson =
```

Comment:
Why do we move this logic to here ? 
`processQueryBuilderLambdaFunction` should have build the queryparameter state